### PR TITLE
chore: custom auth0 variables

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

**Comment by jeysal** [comment]
Hmm, I think TypeScript has a point here saying `process` is not defined, *a lot* of things depend on `@asap-hub/auth` and I wouldn't rely on them inlining env variables as needed.
If we want to keep this centralized, I think perhaps we should go for https://babeljs.io/docs/en/babel-plugin-transform-inline-environment-variables? (I can write typings to go with that change in minutes if we decide for it)

File: packages/auth/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   },
   "devDependencies": {
     "@asap-hub/eslint-config-asap-hub": "workspace:packages/eslint-config-asap-hub",
-    "@auth0/auth0-spa-js": "^1.9.0"
+    "@auth0/auth0-spa-js": "^1.9.0",
```
Comment (jeysal) [inline packages/auth/package.json:19]: tbc back to 12 when rebasing on #211

**Comment by jeysal** [comment]
It seems the Auth0 dev environment isn't configured according to docs yet, can we do that before merge so that signin page etc. already match prod closely? @fampinheiro

File: packages/auth/package.json (hunk 1 above)
Comment (jeysal) [reply packages/auth/package.json:19]: Can rebase on master now and use its something something 12.12 version everywhere :sweat_smile:

**Review by jeysal — APPROVED** [review]
LGTM, as I said I think we should configure the Auth0 dev env as well before merging

**Comment by fampinheiro** [comment]
The only thing missing is the authentication page, the rules are already in place and working :+1:

**Comment by jeysal** [comment]
Okay. Any questions on `auth0.md` lmk

**Pull request merged**